### ISSUE TITLE
Use child-frame or pos-tip?

### ISSUE THREAD

**Issue by seagle0128** [redguardtoo/vc-msg#5]
`popup` is obsolete and buggy. Its advantage is only displaying on bot GUI and terminal. Can we use other alternatives in GUI for better experience, like `child-frame`, `pos-frame`, or `lv`?

**Comment by redguardtoo** [comment]
I can make it configurable to use other GUI. Let me think about it.

**Comment by seagle0128** [comment]
That would be good!

I did it myself for now. Please refer to https://github.com/seagle0128/.emacs.d/blob/master/lisp/init-vcs.el#L91.

**Comment by akirak** [comment]
I created a custom fork of vc-msg which uses pretty-hydra. It partly depends on magit, but I can contribute other parts of the code:

https://github.com/akirak/vc-msg/blob/8c76b3ce7ba497477c8fae69d7b157efaed74973/vc-msg.el#L298

![image](https://user-images.githubusercontent.com/6270544/70627634-ece81d00-1c69-11ea-9def-dfc373e0a6a6.png)

**Comment by seagle0128** [comment]
@akirak It's another good option! 👍 But I have to say it's not necessary to fork to make it.

BTW, are you using `hydra-posframe`?

**Comment by akirak** [comment]
@seagle0128 Yes, I am using hydra-posframe. Hydra makes this kind of task very easy.

**Comment by redguardtoo** [comment]
@akirak , I will try to add your code into sample setup for README.
It's not  necessary to create a fork because this program is totally configurable, see `vc-msg-git-extra`.

**Comment by akirak** [comment]
@redguardtoo 
Thanks, it's a fork because it was easier to hack the package in that way. It probably didn't have to be a fork, but actually I added some features to the original codebase.

Maybe I'll add a PR for those stuffs. Do you want a commiter and commiter date from Git repos?

**Comment by seagle0128** [comment]
The screenshot I customized with `git-messenger` today. Similar with `vc-msg`.

![image](https://user-images.githubusercontent.com/140797/70693843-37fd4100-1cf9-11ea-82b7-d726051a50dc.png)
